.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
what's this underscore?

Reply:
I don't know, that was generated by the husky 4->6 upgrade script

Reply:
That's because now Husky create a `.husky/_/husky.sh` file.
Related: https://blog.typicode.com/husky-git-hooks-javascript-config/